Metatdata Review - METBK

snwhite opened the pull request:
Updates to METBK cal files as a part of the critical metadata review.  Also had to add 2 new METBK UIDs for unknown logger S/Ns.

snwhite commented:
The METBK_Cal_Updates_2019-06-28.xlsx spreadsheet contains the information that was consolidated to update these CSVs.  METBK Heights are in Vault in Sensor Notebook/METBK/METBK_Module_Heights.xlsx.
[METBK_Cal_Updates_2019-06-28.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/3340472/METBK_Cal_Updates_2019-06-28.xlsx)

asmith-whoi commented on calibration/METBKA/CGINS-METLGR-00091__20141213.csv:
```diff
@@ -0,0 +1,9 @@
+LGR033,CC_jcool,1,Constant
+LGR033,CC_jwarm,1,Constant
+LGR033,CC_zinvpbl,600,Constant
+LGR033,CC_use_velpt,1,
```
Logger SN should match filename, filename SN does not match Bulk Load SN CGINS-METLGR-99991, CC_height_of_air_temperature_measurement_m should be 4.255

rtravis-whoi replied:
Agree w Allen - assume that this UID should match the addition in sensor bulk load & should be METLGR-99991; filename and S/N column in csv need to be fixed; fix height of air temp cal coeff.

asmith-whoi commented on calibration/METBKA/CGINS-METLGR-00091__20150507.csv:
```diff
@@ -0,0 +1,9 @@
+LGR033,CC_jcool,1,Constant
+LGR033,CC_jwarm,1,Constant
+LGR033,CC_zinvpbl,600,Constant
+LGR033,CC_use_velpt,1,
```
Logger SN should match filename, filename SN does not match Bulk Load SN CGINS-METLGR-99991, CC_height_of_air_temperature_measurement_m should be 4.255, CP04OOSM should be CP04OSSM

rtravis-whoi replied:
Agree w Allen - assume that this UID should match the addition in sensor bulk load & should be METLGR-99991; filename and S/N column in csv need to be fixed; fix height of air temp cal coeff; fix mooring id.

asmith-whoi commented on calibration/METBKA/CGINS-METLGR-00092__20141213.csv:
```diff
@@ -0,0 +1,9 @@
+LGR033,CC_jcool,1,Constant
+LGR033,CC_jwarm,1,Constant
+LGR033,CC_zinvpbl,600,Constant
+LGR033,CC_use_velpt,1,
```
Logger SN should match filename, filename SN does not match Bulk Load SN CGINS-METLGR-99992, CC_height_of_air_temperature_measurement_m should be 4.255

rtravis-whoi replied:
Agree w Allen - assume that this UID should match the addition in sensor bulk load & should be METLGR-99992; filename and S/N column in csv need to be fixed; fix height of air temp cal coeff.

asmith-whoi commented on calibration/METBKA/CGINS-METLGR-00092__20150507.csv:
```diff
@@ -0,0 +1,9 @@
+LGR033,CC_jcool,1,Constant
+LGR033,CC_jwarm,1,Constant
+LGR033,CC_zinvpbl,600,Constant
+LGR033,CC_use_velpt,1,
```
Logger SN should match filename, filename SN does not match Bulk Load SN CGINS-METLGR-99992, CC_height_of_air_temperature_measurement_m should be 4.255

rtravis-whoi replied:
Agree w Allen - assume that this UID should match the addition in sensor bulk load & should be METLGR-99992; filename and S/N column in csv need to be fixed; fix height of air temp cal coeff.

asmith-whoi commented on bulk/sensor_bulk_load-AssetRecord.csv:
```diff
@@ -1629,6 +1629,8 @@ CGINS-METLGR-00035,,Sensor,0,Bulk Meteorology Instrument Package: METBK LGR Modu
 CGINS-METLGR-00037,,Sensor,0,Bulk Meteorology Instrument Package: METBK LGR Module,METBK LGR MODULE,Star Engineering,ASIMET,LGR037,,,,,Woods Hole Oceanographic Institution
 CGINS-METLGR-00038,,Sensor,0,Bulk Meteorology Instrument Package: METBK LGR Module,METBK LGR MODULE,Star Engineering,ASIMET,LGR038,,,,,Woods Hole Oceanographic Institution
+CGINS-METLGR-99991,,Sensor,0,Bulk Meteorology Instrument Package: METBK LGR Module,METBK LGR MODULE,Star Engineering,ASIMET,N/A,,,,Placeholder for unknown S/N (METBK 1), Woods Hole Oceanographic Institution
+CGINS-METLGR-99992,,Sensor,0,Bulk Meteorology Instrument Package: METBK LGR Module,METBK LGR MODULE,Star Engineering,ASIMET,N/A,,,,Placeholder for unknown S/N (METBK 2), Woods Hole Oceanographic Institution
```
CGINS-METLGR-99991 and CGINS-METLGR-99992 do not match the SN's used on the cal csv's. Which are correct?

rtravis-whoi replied:
I believe the -9999x UIDs are to be used when the METLGR S/N was not captured/documented.

asmith-whoi replied:
The csv's though, are named -00091 and -00092. Should they match what's in this file?

snwhite replied:
The UIDS should be CGINS-METLGR-99991 and CGINS-METLGR-99992.  I was going to do just one (CGINS-METLGR-99999) but because some of the CSV are values depending on if the unit is METBK1 or METBK2, I need to discriminate between them.

rtravis-whoi commented on calibration/METBKA/CGINS-METLGR-00037__20171028.csv:
```diff
@@ -0,0 +1,9 @@
+LGR033,CC_jcool,1,Constant
+LGR033,CC_jwarm,1,Constant
+LGR033,CC_zinvpbl,600,Constant
+LGR033,CC_use_velpt,1,
```
Agree w Allen - fix S/N column in csv to LGR037; fix height of air temp cal coeff.

rtravis-whoi commented on calibration/METBKA/CGINS-METLGR-00037__20181025.csv:
```diff
@@ -0,0 +1,9 @@
+LGR033,CC_jcool,1,Constant
+LGR033,CC_jwarm,1,Constant
+LGR033,CC_zinvpbl,600,Constant
+LGR033,CC_use_velpt,1,
```
Agree w Allen - fix S/N column in csv to LGR037; fix height of air temp cal coeff.

rtravis-whoi commented on calibration/METBKA/CGINS-METLGR-00038__20170606.csv:
```diff
@@ -0,0 +1,9 @@
+LGR033,CC_jcool,1,Constant
+LGR033,CC_jwarm,1,Constant
+LGR033,CC_zinvpbl,600,Constant
+LGR033,CC_use_velpt,1,
```
Agree w Allen - fix S/N column in csv to LGR038; fix height of air temp cal coeff.

rtravis-whoi commented on calibration/METBKA/CGINS-METLGR-00038__20180424.csv:
```diff
@@ -0,0 +1,9 @@
+LGR033,CC_jcool,1,Constant
+LGR033,CC_jwarm,1,Constant
+LGR033,CC_zinvpbl,600,Constant
+LGR033,CC_use_velpt,1,
```
Agree w Allen - fix S/N column in csv to LGR038; fix height of air temp cal coeff.

rtravis-whoi commented on calibration/METBKA/CGINS-METLGR-00038__20190405.csv:
```diff
@@ -0,0 +1,9 @@
+LGR033,CC_jcool,1,Constant
+LGR033,CC_jwarm,1,Constant
+LGR033,CC_zinvpbl,600,Constant
+LGR033,CC_use_velpt,1,
```
Agree w Allen - fix S/N column in csv to LGR038; fix height of air temp cal coeff.

snwhite commented:
@rtravis-whoi @asmith-whoi  I have fixed the issues you identified, and also corrected all of the use_velpt values to 0 (since we do not have a VELPT on the bottom of the buoy).

asmith-whoi commented on calibration/METBKA/CGINS-METLGR-00021__20150815.csv:
```diff
@@ -1,7 +1,7 @@
+LGR021,CC_height_of_air_temperature_measurement_m,5.255,
 LGR021,CC_height_of_windspeed_sensor_above_sealevel_m,5.74,
 LGR021,CC_jcool,1,Constant
 LGR021,CC_jwarm,1,Constant
```
OK

asmith-whoi commented on calibration/METBKA/CGINS-METLGR-00022__20151214.csv:
```diff
@@ -1,8 +1,8 @@
+LGR022,CC_height_of_windspeed_sensor_above_sealevel_m,5.74,
 LGR022,CC_jcool,1,Constant
 LGR022,CC_jwarm,1,Constant
 LGR022,CC_zinvpbl,600,Constant
```
OK

asmith-whoi commented on calibration/METBKA/CGINS-METLGR-00008__20160528.csv:
```diff
@@ -1,9 +1,9 @@
-LGR008,CC_height_of_air_humidity_measurement_m,4.293,
-LGR008,CC_height_of_air_temperature_measurement_m,4.293,
-LGR008,CC_height_of_windspeed_sensor_above_sealevel_m,4.75,
+LGR008,CC_depth_of_conductivity_and_temperature_measurements_m,1.067,CP03ISSM-00004
```
Line 2, depth CT should be 0.98

asmith-whoi commented on calibration/METBKA/CGINS-METLGR-00023__20151021.csv:
```diff
@@ -1,9 +1,9 @@
 LGR023,CC_jwarm,1,Constant
 LGR023,CC_zinvpbl,600,Constant
-LGR023,CC_use_velpt,1,
+LGR023,CC_use_velpt,0,
```
OK

asmith-whoi commented on calibration/METBKA/CGINS-METLGR-00024__20151114.csv:
```diff
@@ -1,8 +1,8 @@
+LGR024,CC_height_of_windspeed_sensor_above_sealevel_m,5.74,
 LGR024,CC_jcool,1,Constant
 LGR024,CC_jwarm,1,Constant
 LGR024,CC_zinvpbl,600,Constant
```
OK

asmith-whoi commented on calibration/METBKA/CGINS-METLGR-00025__20151114.csv:
```diff
@@ -1,8 +1,8 @@
+LGR025,CC_height_of_windspeed_sensor_above_sealevel_m,5.74,
 LGR025,CC_jcool,1,Constant
 LGR025,CC_jwarm,1,Constant
 LGR025,CC_zinvpbl,600,Constant
```
OK

asmith-whoi commented on calibration/METBKA/CGINS-METLGR-00026__20151214.csv:
```diff
@@ -1,8 +1,8 @@
+LGR026,CC_height_of_windspeed_sensor_above_sealevel_m,5.74,
 LGR026,CC_jcool,1,Constant
 LGR026,CC_jwarm,1,Constant
 LGR026,CC_zinvpbl,600,Constant
```
OK

asmith-whoi commented on calibration/METBKA/CGINS-METLGR-00027__20161125.csv:
```diff
@@ -1,8 +1,8 @@
+LGR027,CC_height_of_windspeed_sensor_above_sealevel_m,5.74,
 LGR027,CC_jcool,1,Constant
 LGR027,CC_jwarm,1,Constant
 LGR027,CC_zinvpbl,600,Constant
```
OK

asmith-whoi commented on calibration/METBKA/CGINS-METLGR-00031__20151023.csv:
```diff
@@ -1,8 +1,8 @@
+LGR031,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
 LGR031,CC_jcool,1,Constant
 LGR031,CC_jwarm,1,Constant
 LGR031,CC_zinvpbl,600,Constant
```
OK

asmith-whoi commented on calibration/METBKA/CGINS-METLGR-00032__20151023.csv:
```diff
@@ -1,8 +1,8 @@
+LGR032,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
 LGR032,CC_jcool,1,Constant
 LGR032,CC_jwarm,1,Constant
 LGR032,CC_zinvpbl,600,Constant
```
OK

asmith-whoi commented on calibration/METBKA/CGINS-METLGR-00033__20160513.csv:
```diff
@@ -1,8 +1,8 @@
+LGR033,CC_height_of_windspeed_sensor_above_sealevel_m,4.74,
 LGR033,CC_jcool,1,Constant
 LGR033,CC_jwarm,1,Constant
 LGR033,CC_zinvpbl,600,Constant
```
> 
> 
> CC_height_of_air_temperature_measurement_m should be 4.255

OK

asmith-whoi commented on calibration/METBKA/CGINS-METLGR-00034__20161027.csv:
```diff
@@ -1,8 +1,8 @@
+LGR034,CC_height_of_windspeed_sensor_above_sealevel_m,5.74,
 LGR034,CC_jcool,1,Constant
 LGR034,CC_jwarm,1,Constant
 LGR034,CC_zinvpbl,600,Constant
```
OK

asmith-whoi commented on calibration/METBKA/CGINS-METLGR-00035__20161027.csv:
```diff
@@ -1,8 +1,8 @@
+LGR035,CC_height_of_windspeed_sensor_above_sealevel_m,5.74,
 LGR035,CC_jcool,1,Constant
 LGR035,CC_jwarm,1,Constant
 LGR035,CC_zinvpbl,600,Constant
```
OK

asmith-whoi commented on calibration/METBKA/CGINS-METLGR-00036__20161125.csv:
```diff
@@ -1,8 +1,8 @@
+LGR036,CC_height_of_windspeed_sensor_above_sealevel_m,5.74,
 LGR036,CC_jcool,1,Constant
 LGR036,CC_jwarm,1,Constant
 LGR036,CC_zinvpbl,600,Constant
```
OK
